core/PaginatedList.php

```diff
@@ -457,4 +474,30 @@ public function getRequest() {
 		return $this->request;
 	}
 
+	/**
+	 * Remove the first page pagination get var to stop duplicate page issues
+	 * / and /?start=0 return the same page
+	 *
+	 * @param  string $link
+	 * @return string
+	 */
+	private function removeFirstPagePaginationParam($link)
```

Comment:
I think this function can be implemented in a more simple manner. 

Why not use the inbuilt query string parser and then just under the pagination get var if it's 0

This function could be more generic and called "clean url params" and always run without checking that start is =0 in other functions first